todo/src/App.js

```diff
@@ -31,7 +42,7 @@ class App extends Component {
 const mapStateToProps = state => {
   return {
     todos: state.todos
-  }
-}
+  };
+};
 
-export default connect(mapStateToProps)(App);
+export default connect(mapStateToProps, {addItem:addItem})(App);
```

Comment:
I notice that you're calling the `connect` function on the `<App />` component and passing down the mapped todos as a prop to the `<TodoList />` component. I suggest calling the `connect` method on the `<TodoList />` component from within `TodoList.js` since that is the component that is concerned with the actual todos and `<App />` isn't making use of it.